src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/factory/ApprovedPremisesApplicationEntityFactory.kt

```diff
@@ -171,7 +174,14 @@ class ApprovedPremisesApplicationEntityFactory : Factory<ApprovedPremisesApplica
   }
 
   fun withIsEsapApplication(isEsapApplication: Boolean?) = apply {
```

Comment:
I was going to suggest deprecating this method but I think it's probably worth just removing it and updating the callers to set status instead. Particularly because calling this with 'false' doesn't actually do anything, which could be confusing?

Reply:
This has been removed.